dbos/_sys_db.py

```diff
@@ -231,6 +231,7 @@ def __init__(
         engine_kwargs: Dict[str, Any],
         sys_db_name: Optional[str] = None,
         debug_mode: bool = False,
+        client_mode: bool = False,
```

Comment:
I hate this arg name, but I can't think of a better one.

Reply:
Luckily this is internal facing.